src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,612 @@
+~~~ shell
+kubectl get pods
+~~~
+
+~~~
+NAME                                          READY   STATUS      RESTARTS      AGE
+lms-molt-lms-5cbd7c748-bzbgn                  1/1     Running     0             2m30s
+lms-molt-lms-5cbd7c748-g5zqf                  1/1     Running     0             2m34s
+lms-molt-lms-5cbd7c748-lbjjz                  1/1     Running     0             2m3
+lms-molt-lms-orchestrator-596d4b54d8-c4v97    1/1     Running     0             21m
+...
+~~~
+
+You will see `lms-molt-lms` pods that match the configured [number of LMS instances](#lms-instances), along with one `lms-molt-lms-orchestrator` pod.
+
+## Configuration
+
+To configure the LMS, override the values defined in the [Helm chart](#installation). For information on setting Helm chart values, see the [Helm documentation](https://helm.sh/docs/helm/helm_upgrade/).
+
+{{site.data.alerts.callout_info}}
+Values that are specified in both `lms` and `orchestrator` must have the identical setting.
+{{site.data.alerts.end}}
+
+This section describes the most important and commonly used values.
+
+#### Source dialect
+
+~~~ yaml
+lms:
+  sourceDialect: ""
+...
+orchestrator:
+  sourceDialect: ""
+~~~
+
+You **must** provide a string value for `sourceDialect`, which specifies the dialect of your source database. Supported dialects are:
+
+- `postgres`: PostgreSQL
+- `mysql`: MySQL
+- `cockroach`: CockroachDB
+
+#### Shadowing
+
+~~~ yaml
+lms:
+  shadowMode: none
+~~~
+
+`lms.shadowMode` specifies the shadowing behavior used by the LMS. This should depend on your specific migration requirements. For details, see [Shadowing modes](#shadowing-modes).
+
+#### LMS instances
+
+~~~ yaml
+lms:
+  replicaCount: 3
+~~~
+
+`lms.replicaCount` determines the number of LMS instances created as `lms-molt-lms` pods on the Kubernetes cluster, across which application traffic is distributed. This defaults to `3`.
+
+#### Source and target connections
+
+The connections to the source database and CockroachDB are defined with the configuration keys:
+
+- `INIT_SOURCE`: Connection string for the source database.
+- `INIT_TARGET`: Connection string for a CockroachDB database.
+
+Include client authentication details in the connection strings. For more details, see [Configure source and target certificates](#configure-source-and-target-certificates).
+
+{{site.data.alerts.callout_success}}
+For details about writing a CockroachDB connection string, see [Connect using a URL]({% link {{ page.version.version }}/connection-parameters.md %}#connect-using-a-url).
+{{site.data.alerts.end}}
+
+You should specify the config keys in an external Kubernetes secret and inside a JSON object. The JSON **must** be named `config.json`.
+
+For example:
+
+~~~ json
+config.json: |
+  {
+    "INIT_SOURCE": "mysql://{username}:{password}@{protocol}({host}:{port})/{database}?sslmode=verify-full?sslrootcert=path/to/mysql.ca&sslcert=path/to/mysql.crt&sslkey=path/to/mysql.key",
```

Comment:
We can omit the protocol part and use conn strings for mysql that look like. 
mysql://admin:admin@127.0.0.1:3306/defaultdb. They can basically look like a crdb conn string.